doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_ckan_sink.md

```diff
@@ -434,13 +441,29 @@ Finally, it must be said this way of mapping geolocated context information into
 
 [Top](#top)
 
+####<a name="section2.3.5"></a>About truncating resources
```

Comment:
Probably to late to change... but if I'm understanding correctly the functionality not sure if "truncatnig" is the best name for it. I mean, truncating is about inserting part of the data/record/whatever instead the whole thing. What I understand you are implementing is a kind of "windowing" functionality, either based on size or time.

Or maybe I'm not understanding the functionality :). To be discussed at skype.